config/settings.yml

```diff
@@ -912,6 +912,13 @@
   :allow_api_service_ordering: true
 :prototype:
   :messaging_type: miq_queue
+:rate_limiting:
```

Comment:
❤️  

Thinking out loud - I wonder if this should go into the  `:server` section instead of its own section?  Alternatively, I wonder if we should have different values for UI and API separately.

Reply:
> Thinking out loud - I wonder if this should go into the `:server` section instead of its own section?

Yeah, I don't mind if this is under that section.

> Alternatively, I wonder if we should have different values for UI and API separately.

Maybe?  I don't see why we can't, was just trying to make it simple to start, but if we to split it up from the get go, not too much work to do that.

Reply:
Yeah we can change it later.

Reply:
Changed it now since it wasn't a big deal.